environment-setup.md

```diff
@@ -0,0 +1,152 @@
+# 💻 Setting up your development environment
+
+There are many guides to setting up your computer for development. Each has its own merits. At DataMade, we perform a mix of maintaining legacy projects and developing new projects, often in parallel. Our toolkit is therefore optimized for managing many, isolated versions of packages and getting up and running on new projects quickly.
+
+#### Contents
+
+- [Version control](#version-control)
+- [Text editor](#text-editor)
+- [Python](#python)
+- [Docker](#docker)
+- [Security](#security)
+- [Data](#data)
+- [Geospatial data](#geospatial-data)
+- [Static sites](#static-sites)
+
+For Linux users, most of these tools are available as aptitude packages. When possible, prefer binary packages to building from source. For Mac users, most of these tools are available as [Homebrew](http://brew.sh/) packages. Homebrew is by far the easiest way to manage packages on OS X.
```

Comment:
This guide currently privileges Linux, but MacOS is now the de facto standard for DataMade. (Also, not all Linux distros use aptitude as their package manager.) I think we can comfortably remove the Linux references, since you have to know how to install packages to run Linux anyway.

```suggestion
When possible, we prefer installing binaries from a package manager to building packages from source. For Mac users, most of these tools are available as [Homebrew](http://brew.sh/) packages. Homebrew is by far the easiest way to manage packages on MacOS.
```